### PR TITLE
style quick settings buttons like zoom buttons

### LINKED ISSUE #11907: Bottom navigation vs. map buttons in light mode
### Tell us your idea!

When using the map in light mode, the bottom navigation and bottom-aligned map buttons (quick settings / individual route settings), are adjacent to each other, with no visual separation:

grafik

### What solution would you suggest?

I propose to add some padding to the two map buttons for a better visual separation, just a few dp.

### Is there an alternative?

Leave it as is.

### Additional Information

_No response_

### LINKED ISSUE #12041: outline on map buttons
The two buttons at the bottom of the map have some outline around them that no other button has (at least I couldn't find any). What's the point of that?

imho it looks quite strange, the outline should be removed, those buttons are the same as the zoom buttons and they don't have any border either.
image

@moving-bits you implemented this?

### PR DESCRIPTION
**Pull request by ztNFny**
Use ImageButton instead of Button to remove outlines on map settings buttons

image

## Related issues
fixes #12041 
fixes #11907

### CODE REVIEW

**Comment by bekuno** [comment]
retest this please

File: main/src/cgeo/geocaching/utils/EmojiUtils.java [hunk 1]
```diff
@@ -90,7 +90,8 @@
             /* warning */       0x26a0, 0x26d4, 0x1f6ab, 0x1f6b3, 0x1f6d1, 0x2622,
             /* av-symbol */     0x1f505, 0x1f506,
             /* other-symbol */  0x2b55, 0x2705, 0x2611, 0x2714, 0x2716, 0x2795, 0x2796, 0x274c, 0x274e, 0x2733, 0x2734, 0x2747, 0x203c, 0x2049, 0x2753, 0x2757,
-            /* flags */         0x1f3c1, 0x1f6a9, 0x1f3f4, 0x1f3f3
+            /* flags */         0x1f3c1, 0x1f6a9, 0x1f3f4, 0x1f3f3,
+            /* numbers */       0x31, 0x32, 0x33
```
Comment (moving-bits) [inline main/src/cgeo/geocaching/utils/EmojiUtils.java:94]: This is some different change - introducing numbers as emojis?
Should be a separate PR, and (IIRC) should go from 0 to 9 to cover ten stage lab adventures as well.

**Comment by ztNFny** [comment]
retest this please

File: main/src/cgeo/geocaching/maps/CGeoMap.java [hunk 2]
```diff
@@ -771,7 +771,7 @@ public boolean onPrepareOptionsMenu(@NonNull final Menu menu) {
 
             menu.findItem(R.id.menu_as_list).setVisible(!isLoading() && caches.size() > 1);
 
-            getIndividualRouteUtils().onPrepareOptionsMenu(menu, activity.findViewById(R.id.container_individualroute), individualRoute, StringUtils.isNotBlank(targetGeocode) && null != lastNavTarget);
+            //getIndividualRouteUtils().onPrepareOptionsMenu(menu, activity.findViewById(R.id.container_individualroute), individualRoute, StringUtils.isNotBlank(targetGeocode) && null != lastNavTarget);
```
Comment (moving-bits) [inline main/src/cgeo/geocaching/maps/CGeoMap.java:774]: Why did you comment out this line?

**Review by moving-bits — CHANGES_REQUESTED** [review]

**Comment by ztNFny** [comment]
oops, had the wrong upstream configured, fixed now

**Review by moving-bits — APPROVED** [review]
LGTM, thanks for fixing the issues

**Comment by bekuno** [comment]
retest this please

**Comment by bekuno** [comment]
Looks like a bigger problem on CI:
~~~
Installing APK 'cgeo-debug.apk' on 'android-30(AVD) - 11' for main:basic-debug
Unable to install /home/jenkins/slave/workspace/cgeo CI - PR build/main/build/outputs/apk/basic/debug/cgeo-debug.apk
com.android.ddmlib.InstallException: Unknown failure: Exception occurred while executing 'install':
java.lang.IllegalArgumentException: Error: Failed to parse APK file: /data/local/tmp/cgeo-debug.apk: Failed to parse /data/local/tmp/cgeo-debug.apk
~~~
@mucek4 Please restart the executor(s).

**Comment by moving-bits** [comment]
retest this please

**Comment by moving-bits** [comment]
retest this please

**Comment by ztNFny** [comment]
retest this please

File: main/res/drawable/white_bcg.xml [hunk 3]
```diff
@@ -4,8 +4,8 @@
     android:color="@android:color/black">
     <item>
         <shape android:shape="rectangle">
-            <solid android:color="#C0FFFFFF" />
-            <corners android:radius="2dp" />
+            <solid android:color="@color/osm_zoomcontrolbackground" />
+            <corners android:radius="2dp" />/>
```
Comment (moving-bits) [inline main/res/drawable/white_bcg.xml:8]: somehow the closing tag got duplicated

**Review by moving-bits — CHANGES_REQUESTED** [review]

**Review by moving-bits — APPROVED** [review]
thanks for fixing the duplication

**Pull request merged**